CI/codefresh.yml

```diff
@@ -155,6 +155,17 @@ steps:
           - TWEEK_API_URL=http://api/
           - GIT_PRIVATE_KEY_PATH=/gitkeys/tweekgit
         command: /bin/bash -c "wget --tries 20 --timeout=15 --read-timeout=20 --waitretry=30 --retry-connrefused http://api/status && yarn test -- --host selenium --port 4444"
+      authoring-tests:
+        image: ${{BuildingTests}}
+        volumes_from:
+          - keys
+        depends_on:
+          - authoring
+        environment:
```

Comment:
i think you need to add `api` to the depends_on list, because if you don't it won't be able to connect to the api container. not sure though.